src/components/ImageButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { useNavigation } from '@react-navigation/native';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import { Icon, colors, texts } from '../config';
+
+import { Button } from './Button';
+import { Wrapper } from './Wrapper';
+
+type TImageButton = {
+  icon?: React.ReactElement;
+  params?: any;
+  routeName: string;
+  title?: string;
+};
+
+export const ImageButton = ({ button }: { button: TImageButton }) => {
+  const { icon, params, routeName, title = texts.sue.homeCarousel.button } = button;
```

Comment:
i suggest not having a fallback title. we save some code here an in texts.js.
the button text should be passed in any case.

Reply:
Yes, that's right. I'll change that.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/957/commits/8a32da27e4cd615ba5eae900256f46644863e75d